assets/css/style.css

```diff
@@ -0,0 +1,123 @@
+@import url('https://fonts.googleapis.com/css?family=Cabin&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Bangers&display=swap');
+
+body {
+    color: whitesmoke !important;
+    font-family: Cabin;
+    background-image: url('../images/squiggly.jpg') !important;
+    background-color: rgb(255, 162, 115) !important;
+}
+
+h1 {
+    font-size: 30px !important;
+}
+
+h3 {
+    font-size: 25px !important;
+}
+
+h4 {
+    font-size: 18px !important;
+}
+
+header {
+    font-size: 70px;
+    color: #333;
+    padding: 30px 0 0 0;
+    text-align: center;
+    font-family: Bangers;
+    background-color: #EBF1FF;
+    box-shadow: 2px 2px #888888;
+}
+
+#startGame {
+    height: 300px;
+    padding: 100px 30px;
+}
+
+#start {
+    background-color: whitesmoke;
+    width: 100%;
+    border-radius: 10px;
+    padding: 20px;
+
+}
+
+#start>h2 {
+    padding: 10px;
+    font-size: 35px;
+    font-family: 'Courier New', Courier, monospace;
+    font-weight: bolder;
+    margin: 0;
+}
+
+#start:hover {
+    background-color: #FF9C6B;
+    width: 100%;
+    border-radius: 10px;
+    padding: 20px;
+    border: none;
+}
+
+#playGame {
+    display: none;
+}
+
+.correctAnswer {
+    background-color: #FF9C6B;
+    border-radius: 10px;
+    color: #333;
+    border: 1px #757B80 solid;
+    transition: .3s;
+
+}
+
+.container-fluid {
+    background-color: #1B2B33 !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555494)